docs/user/troubleshooting/05-02-resource-cr-missing-from-cluster.md

```diff
@@ -0,0 +1,41 @@
+# Resource CR Missing from the Cluster
+
+## Symptom
+
+Your service instance or binding is visible in SAP BTP, but its corresponding custom resource (CR) is missing in your Kyma cluster.
+
+## Cause
+
+The CR representing an SAP BTP service instance or service binding is absent from the Kyma cluster. This happens if the CR was accidentally deleted or if there are issues preventing its synchronization or visibility in the cluster. While the resource still exists, the connection to your Kyma cluster is broken due to the absence of the CR.
+
+## Solution
+
+To address this issue, you can manually recreate the Kyma CR using details from SAP BTP to restore the connection with the existing BTP resource. For successful reconnection, ensure that the new CR matches the name, resides in the same namespace, and is linked to the same cluster ID as the original CR. Since the SAP BTP resource maintains its configuration, matching these attributes allows the new CR to reconnect to the existing BTP resource without provisioning a new one.
```

Comment:
Is it Kyma CR? Or ServiceInstance/ServiceBinding CR created in the Kyma cluster?

Reply:
Good catch, thank you!